Fastfile.private

```diff
@@ -78,12 +79,12 @@ reflected in `Info.plist` during building."
     ).to_i
 
     bump_type = check_bump_type(
-      build_configuration: build_configuration,
+      build_configuration: build_configuration_key,
```

Comment:
Why the key and not the enum?

Reply:
As it's written it receives the symbol. We can change it.